### DISCUSSION TITLE
Folder blacklist when Trigger Templater on new file creation

### DISCUSSION THREAD

**Discussion by Superkikim** [SilentVoid13/Templater#1252] [Ideas & Feature Requests]
Hi. 

Currently, when Trigger Templater on new file creation is enabled, it will be triggered on any file creating event. 

I suggest you add a folder black list where we can exclude folders where we DON'T want Templater to be triggered. 

This is useful when we use a plugin of a script to import a large amount of file, which do not need Templater to be triggered.

**Comment by Zachatoo** [comment]
Would it make more sense to disable that setting before the import, then re-enable it after you're done?

Is it a common use case to consistently import into the same folder(s), such that it's worth the time to setup blacklist folders instead of disabling the setting?

Is it common to import files that have Templater tags in them? If they don't have Templater tags, nothing should happen.

**Comment by Superkikim** [comment]
That would be cumbersome. It's a plugin that can imports hundreds of
notes in seconds. I thought a black list could be helpful. 

I asked because when I import the notes, I have plenty of message in
the console. And I don't really understand why. But it's most probably
by design. It just says the notes are more than 10000:

Templater skipped parsing
Outils/ChatGPT/Archives/2023/11/e12d89c0-e6cb-470e-8eb7-1d44805b9638.md
because file size exceeds 10000

But I guess, this is just info, and the trigger doesn't add pressure
to the computer resources ?

**Reply by Zachatoo** [reply]
It wouldn't be any more intensive than checking the blacklist, no. So for your use case it would just remove the messages from the console.

If you want it to process faster, disabling Templater would be the way to go, though the processing will be minimal if they're all large files.

**Comment by Superkikim** [comment]
Sounds right. Tx for clarifying.

Have a nice end of week.
